addons/sprite-properties/userscript.js

```diff
@@ -36,11 +36,22 @@ export default async function ({ addon, console, msg }) {
     }
   }
   // Close properties panel when mouse leaves the entire sprite panel
+  let popoverAbove;
   document.body.addEventListener(
-    "mouseleave",
+    "mousemove",
```

Comment:
The event will fire every constantly even when on the project page. Is there a better solution?